ui/raidboss/data/04-sb/raid/o12s.js

```diff
@@ -111,58 +359,80 @@ export default {
       suppressSeconds: 1,
       response: Responses.tankBuster(),
     },
     {
       id: 'O12S Electric Slide Marker',
       netRegex: NetRegexes.headMarker({ id: '009[12345678]' }),
       condition: Conditions.targetIsYou(),
-      alertText: function(data, matches, output) {
-        let num = parseInt(matches.id);
+      response: (data, matches, output) => {
+        // cactbot-builtin-response
+        output.responseOutputStrings = {
+          square: {
+            en: '#${num} Square',
+            de: '#${num} Viereck',
+            fr: '#${num} Carré',
+            ja: '#${num} 四角',
+            cn: '#${num} 四角',
+            ko: '#${num} 짝수',
+          },
+          triangle: {
+            en: '#${num} Triangle',
+            de: '#${num} Dreieck',
+            fr: '#${num} Triangle',
+            ja: '#${num} 三角',
+            cn: '#${num} 三角',
+            ko: '#${num} 홀수',
+          },
+        };
+
+        const id = matches.id;
+        if (!id)
+          return;
+
+        let num = parseInt(id);
         const isTriangle = num >= 95;
         num -= 90;
         if (isTriangle)
           num -= 4;
 
-        return isTriangle ? output.triangle({ num: num }) : output.square({ num: num });
-      },
-      outputStrings: {
-        square: {
-          en: '#${num} Square',
-          de: '#${num} Viereck',
-          fr: '#${num} Carré',
-          ja: '#${num} 四角',
-          cn: '#${num} 四角',
-          ko: '#${num} 짝수',
-        },
-        triangle: {
-          en: '#${num} Triangle',
-          de: '#${num} Dreieck',
-          fr: '#${num} Triangle',
-          ja: '#${num} 三角',
-          cn: '#${num} 三角',
-          ko: '#${num} 홀수',
-        },
+        // Odd numbers have to run to the other side, so make this louder.
+        // TODO: should this be alarm/alert instead of alert/info?
+        const isOdd = num % 2;
+        const text = isTriangle ? output.triangle({ num: num }) : output.square({ num: num });
+        return { [isOdd ? 'alertText' : 'infoText']: text };
       },
     },
     {
       id: 'O12S MF Stack Marker',
       netRegex: NetRegexes.headMarker({ id: '003E', capture: false }),
-      condition: function(data) {
-        return !data.isFinalOmega;
-      },
+      condition: (data) => !data.isFinalOmega,
       suppressSeconds: 1,
-      response: Responses.stackMarker('info'),
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Stack Groups',
+        },
+      },
     },
     {
       id: 'O12S Optimized Meteor',
       netRegex: NetRegexes.headMarker({ id: '0057' }),
       condition: Conditions.targetIsYou(),
-      response: Responses.meteorOnYou('info'),
+      response: Responses.meteorOnYou('alarm'),
+    },
+    {
+      id: 'O12S Optimized Sagittarius Arrow',
+      netRegex: NetRegexes.headMarker({ id: '0017' }),
+      condition: Conditions.targetIsYou(),
+      alertText: (data, _, output) => output.text(),
```

Comment:
```suggestion
      alarmText: (data, _, output) => output.text(),
```

Reply:
Counterpoint: it's good to differentiate meteor and arrow? At least in non-BLU land you do different things.

Reply:
That's fair; I forgot that they're not both just "run away" normally